service-registry/src/main/java/org/apache/servicecomb/serviceregistry/config/ServiceRegistryConfig.java

```diff
@@ -133,10 +131,8 @@ public boolean isSsl() {
     uriList.forEach(anUriList -> {
       try {
         URI uri = new URI(anUriList);
-        StringBuilder sb = new StringBuilder(uri.getHost());
-        sb.append(':').append(uri.getPort() < 0 ? PROTOCOL_HTTP_PORT : uri.getPort());
         this.ssl = uri.getScheme().startsWith("https");
```

Comment:
I think it's better use "https" equals uri.getScheme

Reply:
done